dev/managedservicesplatform/internal/stack/monitoring/monitoring.go

```diff
@@ -149,7 +150,8 @@ func NewStack(stacks *stack.Set, vars Variables) (*CrossStackOutput, error) {
 				// Supress all notifications if opsgenieAlerts is disabled -
 				// this allows us to see the alerts, but not necessarily get
 				// paged by it.
-				SuppressNotifications: pointers.Ptr(!opsgenieAlerts),
+				// TODO: Enable after we dogfood the alerts for a while.
+				SuppressNotifications: pointers.Ptr(false),
```

Comment:
Shouldn't this be true?
([docs](https://registry.terraform.io/providers/opsgenie/opsgenie/latest/docs/resources/api_integration#suppress_notifications))
```suggestion
				SuppressNotifications: pointers.Ptr(true),
```

Reply:
aha!